doc/tutorials/mediasdk-tutorials-readme.md

```diff
@@ -125,7 +124,7 @@ Tutorials are devided into few sections:
 | simple_3_encode_vmem_async | Adds asynchronous operation to previous example, resulting in further improved performance |
 | simple_4_vpp_resize_denoise | Showcases video frame processing (VPP) using system memory surfaces. Highlights frame resize and denoise filter processing |
 | simple_4_vpp_resize_denoise_vmem |  Adds use of video memory surfaces for improved VPP performance |
-| simple_5_transcode_|  Transcodes (decode+encode) AVC stream to another AVC stream using system  memory surfaces|
+| simple_5_transcode |  Transcodes (decode+encode) AVC stream to another AVC stream using system memory surfaces|
```

Comment:
Idea for the next iteration of cleanup: you may add links to the git folders containing these samples.

Reply:
alright